tests/utils/nfts.ts

```diff
@@ -109,3 +131,238 @@ export const mintCollection = async (
 
   return { collection: collectionNft, members: collectionMembers };
 };
+
+export async function createTestMintAndTokenT22VanillaExt(
+  connection: Connection,
+  payer: Keypair,
+  recipient?: PublicKey,
+  groupKeyPair?: Keypair,
```

Comment:
lets make some of these arguments `PublicKey` if there is no need for them to be a keypair